snyk/requiremtnts.txt

```diff
@@ -0,0 +1,2 @@
+snowflake-connector-python<4.0.0
+SQLAlchemy>=1.4.19,<2.0.0
```

Comment:
Do you want add it by hand? I'm not convinced about this. Maybe comment in pyproject.toml `Update requirements.txt too`?

Reply:
I added pre-commit hook based on the one from snowflake-cli